notes.md

```diff
@@ -0,0 +1,14 @@
+To Do:
+
+1. Create home page
+2. Create single coin page
+3. Create search function
+4. create sort functions
+
+Top 100 coins info:
+
+Global coins info:
+https://api.coingecko.com/api/v3/global
+
+questions:
+How do I create one main math folder, and export the functions out of it to be used individually?
```

Comment:
you make a utils folder and create inside files. from those files you can export functions and variables like you would with a React component